src/services/express-service.js

```diff
@@ -10,9 +10,9 @@ const log = getActiveLogger();
  * adding express as a dependency.
  * @returns {Promise<void>}
  */
-export async function generateExpressFiles(projectName) {
+export async function generateExpressFiles(projectName, filePath) {
```

Comment:
could we simply pass in the file path, instead of the name and the path? That way we are rewriting the logic to get file path over and over